### DISCUSSION TITLE
GitHub Student Developer Application Rejected

### DISCUSSION THREAD

**Discussion by maliksaqibahmad** [community/community#147872] [GitHub Education]
### Hi everyone,

I have submitted my documents four times, but each time I have been rejected. The primary reason for the rejections seems to be that I am not currently on campus, and I won’t be returning until January 13th, 2025 due to winter vacation. Despite providing an official notification from the government confirming this, my submissions have still been unsuccessful.

**Comment by emansarahafi** [comment]
Hi, I am sorry that you are going through this. I went through this before and the only way to solve this is to be in same city as your campus. If it is urgent, you can open a ticket with GitHub Support by clicking [here](https://support.github.com/contact/education).

**Reply by maliksaqibahmad** [reply]
I am in the same city. So guess I'll open the ticket and see.
